src/main/java/hudson/matrix/MatrixProject.java

```diff
@@ -36,26 +36,9 @@
 import hudson.init.InitMilestone;
 import hudson.init.Initializer;
 import hudson.matrix.MatrixBuild.MatrixBuildExecution;
-import hudson.model.AbstractProject;
-import hudson.model.Action;
-import hudson.model.BuildableItemWithBuildWrappers;
-import hudson.model.DependencyGraph;
-import hudson.model.Descriptor;
+import hudson.model.*;
```

Comment:
It is a bad practice for production code, because there is a risk of ambiguity

Reply:
Sorry, It's been a mistake. MatrixProject and MatrixConfiguration shouldn't been committed neither pulled.